src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -724,6 +756,87 @@ public OptionData setRequiredRange(double minValue, double maxValue)
         return this;
     }
 
+    /**
+     * Configure the minimum length for strings which can be provided for this option.
+     *
+     * @param  minLength
+     *         The minimum length for strings which can be provided for this option.
+     * @throws IllegalArgumentException
+     *         If any of the following checks fail
+     *         <ul>
+     *             <li>{@link OptionType type of this option} is {@link OptionType#STRING STRING}</li>
+     *             <li>{@code minLength} is greater than or equal to {@code 0}</li>
+     *         </ul>
```

Comment:
Oh I thought we killed this way of defining `IllegalArgumentException`.
It really should be more like
```
@throws IllegalArgumentException
     *         <ul>
     *             <li>If the {@link OptionType type of this option} is not {@link OptionType#STRING STRING}</li>
     *             <li>If {@code minLength} is less than to {@code 0}</li>
     *         </ul>
``` 

I guess we don't _have_ to do it in this PR, but this is the pattern we've moved to in JDA in every other place.

Reply:
Ah, tracked this down. We changed it in a PR that never got merged.
I updated and merged that PR, so this language should be changed to match the new system before we merge.